libclc/amdgcn-amdhsa/libspirv/group/collectives.cl

```diff
@@ -235,16 +235,16 @@ __CLC_GROUP_COLLECTIVE(FAdd, __CLC_ADD, float, 0)
 __CLC_GROUP_COLLECTIVE(FAdd, __CLC_ADD, double, 0)
 
 // There is no Mul group op in SPIR-V, use non-uniform variant instead.
```

Comment:
Please, update the comment as well.
I think it's useful to keep the same updated comment in libclc/ptx-nvidiacl/libspirv/group/collectives.cl.

Reply:
I have removed the comment. I don't think it makes much sense to keep it anymore as the KRH suffix is just to signify that it is an extension. Names like that shouldn't comes as much of a surprise with all the `__spirv_*INTEL` we have. 😄

Reply:
Okay. :)